### PR TITLE
Move sys-container Dockerfiles to a separate repo & update doc pointers [skip ci]

### PR DESCRIPTION
**Pull request by rodnymolina**
Move sys-container Dockerfiles to a separate repo & update doc pointers [skip ci]

Signed-off-by: Rodny Molina <rmolina@nestybox.com>

### CODE REVIEW

File: docs/quickstart/dind.md [hunk 1]
```diff
@@ -224,7 +225,7 @@ manager inside a system container.
 
 We will use a system container image called `nestybox/alpine-supervisord-docker:latest`.
 Nestybox DockerHub public repo. The Dockerfile, supervisord.conf, and docker-entrypoint.sh files
-can be found [here](../../sys-container/dockerfiles/alpine-supervisord-docker).
+can be found [here](https://github.com/nestybox/dockerfiles/tree/main/alpine-supervisord-docker).
```
Comment (ctalledo) [inline docs/quickstart/dind.md:228]: "/tree/main" or "/blob/main" ?

**Review by ctalledo — COMMENTED** [review]
Only one nit ...

File: docs/quickstart/dind.md (hunk 1 above)
Comment (rodnymolina) [reply docs/quickstart/dind.md:228]: Github by default makes use of 'tree' objects to reference directories, and it relies on 'blobs' to refer to files. This link is correct.

Comment (ctalledo) [reply docs/quickstart/dind.md:228]: Ok got it.

**Review by ctalledo — APPROVED** [review]

**Pull request merged**